lib/blocks.php

```diff
@@ -164,11 +164,32 @@ function gutenberg_render_block( $block ) {
 		}
```

Comment:
Line 163 above will also need to be adjusted to pass the content along -- as at the moment it doesn't.

Reply:
I think ideally the function should be revised to something like --

```php
/**
 * Renders a single block into a HTML string.
 *
 * @since 1.9.0
 *
 * @param  array $block A single parsed block object.
 * @param  array $inner_blocks A list of processed inner blocks.
 * @return string String of rendered HTML.
 */
function gutenberg_render_block( $block, $inner_blocks = [] ) {
	$block_name = isset( $block['blockName'] ) ? $block['blockName'] : null;
	$block_type = $block_name ? WP_Block_Type_Registry::get_instance()->get_registered( $block_name ) : null;
	$attributes = is_array( $block['attrs'] ) ? $block['attrs'] : array();

	// If there are innerBlocks, stitch their results back in.
	if ( ! empty( $block['blockMarkers' ] ) ) {
		$output = '';
		$index = 0;
		foreach( $block['blockMarkers'] as $i => $p ) {
			$output .= substr( $block['innerHTML'], $index, $p - $index );
			$output .= $inner_blocks[ $i ];
			$index = $p;
		}
		$output .= substr( $block['innerHTML'], $index );

		$block['innerHTML'] = $output;
	}

	// Handle both dynamic and static.
	if ( null !== $block_type && $block_type->is_dynamic() ) {
		return $block_type->render( $attributes, $block['innerHTML'] );
	}
	return $block['innerHTML'];
}
```